codalab/lib/worksheet_util.py

```diff
@@ -434,6 +434,12 @@ def friendly_render_dep(dep):
             # FIXME(sckoo): we will be passing the old permissions format into this
             # which has been updated to accommodate the new formatting
             return group_permissions_str(bundle_info['group_permissions'])
+    elif genpath == 'owner_name':
+        from bottle import local
```

Comment:
Imports are better put on top of file, see https://www.python.org/dev/peps/pep-0008/#imports